apps/docs/src/components/HomeLayout/Content/index.tsx

```diff
@@ -41,7 +36,7 @@ export default Content;
 const styles = StyleSheet.create({
 	container: {
 		paddingTop: 40,
-		gap: 30,
-		alignItems: 'center',
+		maxWidth: 1000,
```

Comment:
should be something like 1024 or 1280 or 1620 I think, please check which number is that in our Home (Landing page) and use same value here -- should put this `maxWidth` in a reuse-able module like `utils/config.ts`